File: Core Java/interface-polymorphism/src/com/techlabs/men/Men.java
package com.techlabs.men;

import com.techlabs.imannerable.IMAnnerable;

public class Men implements IMAnnerable {

	@Override
	public void wish() {
		System.out.println("Men is wishing....");
	}

	@Override
	public void depart() {
		System.out.println("Men is upto Depart.....");
		
	}

}

File: DOTNET/C#/InterfacePolymorphismApp/InterfacePolymorphismApp/Men.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterfacePolymorphismApp
{
    public class Men : IMAnneravle
    {
        public void Depart()
        {
            Console.WriteLine("Men is upto Depart.......");
        }

        public void Wish()
        {
            Console.WriteLine("Men is wishing......");
        }
    }
}
